app/controllers/events/routes.py

```diff
@@ -1,18 +1,24 @@
 from flask import request, render_template, g
 from flask import Flask, redirect, flash
-
 from app.controllers.events import events_bp
-from app.logic.events import getEvents
+from app.models.term import Term
+from app.logic.events import groupEventsByCategory
 from app.logic.getUpcomingEvents import getUpcomingEventsForUser
 
-@events_bp.route('/events', methods=['GET'])
-def events():
-    events = getEvents()
+@events_bp.route('/events/<term>/', methods=['GET'])
+def events(term):
+
+    eventsDict = groupEventsByCategory(term)
+    print(eventsDict)
```

Comment:
Don't leave temporary debug output in the code

Reply:
Removed line 12